x/staking/types/params.go

```diff
@@ -28,6 +28,7 @@ const (
 	// DefaultHistorical entries is 10000. Apps that don't use IBC can ignore this
 	// value by not adding the staking module to the application module manager's
 	// SetOrderBeginBlockers.
+	// Deprecated: HistoricalEntries is deprecated
 	DefaultHistoricalEntries uint32 = 10000
```

Comment:
Why keep this? given that we are already API breaking with the NewParams arg changes